ksp_plugin/orbit_analyser.cpp

```diff
@@ -120,9 +120,8 @@ Status OrbitAnalyser::AnalyseOrbit(GuardedParameters guarded_parameters) {
         parameters.extended_mission_duration.value_or(
             parameters.mission_duration),
         std::max(2 * smallest_osculating_period, parameters.mission_duration));
-    for (Instant t = parameters.first_time + analysis_duration / 0x1p10;
-         trajectory.back().time < parameters.first_time + analysis_duration;
-         t += analysis_duration / 0x1p10) {
+    for (int n = 0; n <= 0x1p10; ++n) {
```

Comment:
Could we invest in a constant for 1024?  This number shows up in two places.

Also, `const` the parameter at line 86 (stupid github won't let me comment there.)